docs/LSP4IJ-Continuous-Integration.md

```diff
@@ -218,6 +218,14 @@ For example, if you want to trigger the Build Workflow from the `main` branch of
 
 <img alt="Example" height="300" src="images/example-manually-trigger.png" width="300" style="display: block; margin: 0 auto;"/>
 
+If you want to check out LSP4IJ based on a PR number, you can check the box labeled `If you checked this, please make sure to enter only the LSP4IJ PR number below` and specify the PR number in the provided text field labeled `Reference/branch for LSP4IJ checkout`. Additionally, you can specify the tag or branch of the `liberty-tools-intellij` repository to be used. 
+
+> NOTE: Make sure to enable the checkbox labeled `Use lsp4ij locally` as well for this case.
+
+For example, if you want to trigger the Build Workflow from a branch of the LTI repository and build against LSP4IJ PR number `500` while using the `main` branch of LTI, you would configure it as follows:
+
+<img alt="Example" height="300" src="images/Allow-developer-to-input-PR-number.png" width="300" style="display: block; margin: 0 auto;"/>
```

Comment:
Update the image to show the new text.

Reply:
I updated the image with the same name

Reply:
@TrevCraw,  I updated the other images in the document and checked the other document files as well, but I couldn't find any images showing the inputs for running the build.yaml.